terraform/modules/monitoring/dashboard.json

```diff
@@ -173,8 +277,122 @@
           }
         },
         "width": 2,
+        "xPos": 0,
+        "yPos": 6
+      },
+      {
+        "height": 2,
+        "widget": {
+          "title": "Cloud Run: Memory usage for Cert Rotation Server",
+          "xyChart": {
+            "chartOptions": {
+              "mode": "COLOR"
+            },
+            "dataSets": [
+              {
+                "minAlignmentPeriod": "60s",
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "apiSource": "DEFAULT_CLOUD",
+                  "timeSeriesFilter": {
+                    "aggregation": {
+                      "alignmentPeriod": "60s",
+                      "crossSeriesReducer": "REDUCE_PERCENTILE_99",
+                      "perSeriesAligner": "ALIGN_SUM"
```

Comment:
do we want to sum up the memory usages? This should probably be 99th percentage or avg

Reply:
I do see the reducer is set to 99th. its not clear to me the difference between reducer and aligner

Reply:
Me neither. I simply used the same parameters that cloud run metrics tab has. I suppose it's the good version.
I also didn't find a way to logically separate metrics. Need more learning. Should we get this in first?

Reply:
Sure, for now we can merge as-is.